uint8 RESUME=0
uint8 PAUSE=1
uint8 toggle_action
---
